pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotRealtimeTableResource.java

```diff
@@ -90,6 +90,23 @@ public Response resumeConsumption(
     }
   }
 
+  @POST
+  @Path("/tables/{tableName}/resetConsumption")
```

Comment:
Suggest making the API more explicit that it is just force committing the consuming segment. Something like `POST /tables/{tableName}/forceCommit`

Reply:
+1 . we already have reset API which does helix disable/enable. this is different in logic, so calling it forceCommit would avoid confusion

Reply:
Valid point. Refactored.